File: PlacementCellBackend/Services/Feedback/Interfaces/IFeedbackOnCompanyService.cs
using PlacementCellBackend.DTOs;

namespace PlacementCellBackend.Services.Feedback.Interfaces
{
    public interface IFeedbackOnCompanyService
    {
        Task<IEnumerable<AlumniFeedBackOnCompanyDTO>> GetAllFeedbacksAsync();
        Task<AlumniFeedBackOnCompanyDTO?> GetFeedbackByIdAsync(string id);
        Task<AlumniFeedBackOnCompanyDTO> CreateFeedbackAsync(AlumniFeedBackOnCompanyCreateDTO feedback);
        Task<bool> UpDateFeedbackAsync(int id, AlumniFeedBackOnCompanyCreateDTO feedback);
        Task<bool> DeleteFeedbackAsync(int id);
    }
}

File: PlacementicsBackend/src/main/java/com/example/placementicsbackend/services/Feedback/Interfaces/IFeedbackOnCompanyService.java
package com.example.placementicsbackend.services.Feedback.Interfaces;

import com.example.placementicsbackend.dtos.AlumniFeedbackOnCompanyCreateDto;
import com.example.placementicsbackend.dtos.AlumniFeedBackOnCompanyDTO;
import java.util.List;

public interface IFeedbackOnCompanyService {

    List<AlumniFeedBackOnCompanyDTO> getAllFeedbacks();

    AlumniFeedBackOnCompanyDTO getFeedbackById(String id);

    AlumniFeedBackOnCompanyDTO createFeedback(AlumniFeedbackOnCompanyCreateDto feedback);

    boolean updateFeedback(int id, AlumniFeedbackOnCompanyCreateDto feedback);

    boolean deleteFeedback(int id);
}
